File: praktikum 3/C/PRAK305-2310817210027-MuhammadErzaRaihan.c
#include <stdio.h>
int main()
{
    int detik , menit , jam , hari, X;

    printf("input\n");
    scanf("%d",&detik);

    menit = (detik % 3600)/60;
    jam = (detik%(24 * 3600))/3600;
    hari = detik/(24 * 3600);
    X = detik % 60;

    if (hari>0)
    {
        printf("\nOutput \n%d hari %02d:%02d:%02d", hari, jam, menit, X);
    }
    else{
        printf("\nOutput \n %02d:%02d:%02d", jam, menit , X);
    }

    return 0;
}

File: praktikum 3/Python/PRAK305-2310817210027-MuhammadErzaRaihan.py
detik = int(input("input\n"))

menit = (detik % 3600) // 60
jam = (detik % (24 * 3600)) // 3600
hari = detik // (24 * 3600)
X = detik % 60

if hari > 0:
    print("\nOutput\n{} hari {:02d}:{:02d}:{:02d}".format(hari, jam, menit, X))
else:
    print("\nOutput\n{:02d}:{:02d}:{:02d}".format(jam, menit, X))
